docs/api-reference/components.md

```diff
@@ -0,0 +1,69 @@
+# Components
+
+## RouterProvider
+
+The `RouterProvider` component holds the Yarr router context and is responsible for passing it down to underlying components. This component should be used at a level higher than the page components. It takes the router object as props and passes it down. 
+
+### Usage
+
+1. Create a router object using a routes object and other options
+
+```jsx
+// routes is an array of objects describing which component
+// to load for which url location.
+const router = createBrowserRouter({
+  routes,
+  awaitComponent: true
+});
+``` 
+2. Pass the router object into `RouterProvider` as prop.
+
+```jsx
+ <RouterProvider router={router}>
+   <App/>          
+ </RouterProvider>
+```
+
+### Link
+
+The Link component allows a user to transition to a different url location within the app. 
+
+#### Usage
+
+Once the `Link` component is imported from `yarr`, it can be used like so. 
+
+```jsx
+import { Link } from "yarr";
+
+export const Navbar = () => (
+  <nav>
+    <Link to="/">Home</Link>
+    <Link to="/continents">Continents</Link>
+  </nav>
+);
+```
+
+## Redirect
+
+The `Redirect` component allows a user to redirect to a different path immediately after the first render.
+
+This is generally useful to secure authenticated pages and redirect the user out if they need to be logged in to access a page.
+
+```jsx
+import {Redirect} from 'yarr'
+
+const AuthenticatedUsersOnlyPage = () => {
+    if(!notAuthenticated){
+        return <Redirect to="/other-route"/>
+    }
+    else {
+        ...
+    }
+}
+```
+
```

Comment:
I recommend we nuke documenting the Redirect component since it's a hold over for Contra's migrationand the idea would be this component wouldn't exist in the future. `redirectRules` and using `useNavigation` in certain cases are more preferred since requesting redirects during the "render" phase can mess with things.